### PR TITLE
Add myself as a codeowner default

### PR DESCRIPTION
**Pull request by kevinkjt2000**

### CODE REVIEW

**Comment by PerchunPak** [comment]
This means that you must review every single PR, even all the dependabot bumps. Are you sure?

**Review by ItsDrike — CHANGES_REQUESTED** [review]
In a codeowners file, the last matching pattern takes precedence (see the [docs](https://github.blog/2017-07-06-introducing-code-owners/)), by putting the catch-all at the bottom of the file, it will override any previous entries. Please move it to the top.

That said, why do you want to be the default codeowner? As @PerchunPak pointed out, you'll be requested to review every PR automatically, and no PR will be mergable until you submit an approving review.

This will almost certainly end up slowing down merging a lot, especially when it comes to smaller PRs, where a review from any of the maintainers (me/PerchunPak) would suffice. When you're the default codeowner, that wouldn't be the case, and your review would become a requirement for merging even just a typo fix / dependabot PRs.

Do you really want to have to review every single new PR?

**Comment by kevinkjt2000** [comment]
I want my approvals to carry the same weight as a code owner. I do not want to hold up every PR though.

**Pull request closed without merging**
